ca/ca.go

```diff
@@ -138,6 +148,7 @@ func (m *caMetrics) noteSignError(err error) {
 type certificateAuthorityImpl struct {
 	capb.UnsafeCertificateAuthorityServer
 	sa           sapb.StorageAuthorityCertificateClient
+	sctService   rapb.SCTProviderClient
```

Comment:
bikeshedding: this name is good and descriptive, but departs from our established naming convention (only config struct members are `FooService`; actual gRPC client objects are just named `foo`).

I think the name is _fine_, but will throw out some other ideas for consideration. Feel free to ignore if you don't like them any better:
- `sctProvider`
- `sctClient`